content/content-strategy/cms/cms-requirements/key-takeaway.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: text/callout
+---
+Your CMS is a support tool for your content strategy. It should ensure that content is made  available in the most helpful way possible. It should also be as easy for you to use as possible.
```

Comment:
Double space between 'made' and 'available'.

Reply:
Steve. Fixed.